releasenotes/notes/diagnoses-json-serializ-fix-10609dc20200dabb.yaml

```diff
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+fixes:
```

Comment:
We don't need a releasenote here since the bug was introduce in the same agent version we're fixing it (releasenotes are aimed to customers).

Reply:
Will know next time, thank you (addressed).

Reply:
@hush-hush now "ci/circleci: release_note — Your tests failed on CircleCI". What should I do? Should I apply changelog/no-changelog label? It is kind of strange to put this label.